project-reports/2024/2024-Q3-Hyperledger-Indy.md

```diff
@@ -0,0 +1,144 @@
+[indy-node-container]: https://github.com/hyperledger/indy-node-container
+[indy-blssignatures-rs]: https://github.com/hyperledger/indy-blssignatures-rs
+[indy-besu]: https://github.com/hyperledger/indy-besu
+
+### **Client Tools**
+
+- [indy-vdr]
+- [indy-shared-rs] -- for AnonCreds, [Hyperledger AnonCreds] should be used
+- [indy-cli-rs] -- new, Indy CLI based on Indy VDR, not Indy SDK
+
+[indy-vdr]: https://github.com/hyperledger/indy-vdr
+[indy-shared-rs]: https://github.com/hyperledger/indy-shared-rs
+[indy-cli-rs]: https://github.com/hyperledger/indy-cli-rs
+[Hyperledger AnonCreds]: https://github.com/hyperledger/anoncreds-rs
+
+### **Specifications**
+
+- [did:indy Specification], [did:indy Specification source repository]
+- [did:indy Networks]
+- [Indy HIPE] (Hyperledger Indy Project Enhancements)
+- [indy-did-method]
+
+[did:indy Specification]: https://hyperledger.github.io/indy-did-method/
+[did:indy Specification source repository]: https://github.com/hyperledger/indy-did-method
+[did:indy Networks]: https://github.com/hyperledger/indy-did-networks
+[Indy HIPE]: https://github.com/hyperledger/indy-hipe
+[indy-did-method]: https://github.com/hyperledger/indy-did-method
+
+## Project Health
+
+The project had less contributor activity over the quarter. Per the [Indy Quarterly
+Activity Dashboard], in the months April through June, 2024 there were 426
+commits (down about 50%) and 77 contributors (down about 30%). Work was focused
+on dependabot updates, the [Indy Besu] efforts (which is not included in the LFX
+data), and client performance improvements in digital credential wallets.
+
+[Indy Quarterly Activity Dashboard]: https://insights.lfx.linuxfoundation.org/foundation/lf-decentralized-trust/overview/github?project=indy&routedFrom=Github&bestPractice=false&dateFilters=Last%20Quarter&dateRange=2024-04-01%20to%202024-06-30&compare=PP&granularity=week&hideBots=true
+
+## Questions/Issues for the TOC
+
+None
+
+### Issues from previous reports
+
+#### **Diversity of Contributor Community**
+
+See the updated type of information in the appropriate section of this report.
+
+## Releases
+
+Release since the last quarterly report -- those after September 30, 2023:
+
+- indy-node - None
+- indy-plenum - v1.14.0rc0 -- first release candidate for the Ubuntu 22.04 release of Indy Plenum
+- indy-vdr -  - v0.4.2, v0.4.3
+- indy-node-container - v1.2.6
+- indy-shared-rs - None
+
+## Overall Activity in the Past Quarter
+
+Work continued this quarter on the Ubuntu 22.04 version of Indy.
+
+The [indy-vdr] work focused on improving performance in Wallets, and especially
+caching immutable transactions.
+
+Progress was made on the [Indy Besu] implementation of Indy capabilities on
+a Besu base, and in defining the Indy Besu updates to the [Indy DID Method].
+
+[Indy Besu]: https://github.com/hyperledger/indy-besu
+[Indy DID Method]: https://github.com/hyperledger/indy-did-method
```

Comment:
We can also note that work to implement the revocation feature is underway right now.